src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -192,11 +182,130 @@ public boolean rightLowerLimitReached() {
 
     @Override
     public void periodic() {
+        if (enabled) {
+            double output = verticalController.calculate(getLeftPosition());
+            double error = errorController.calculate(DriveTrain.getInstance().getPitch());
+
+            double power = MathUtil.clamp(output, LiftConstants.backwardsMotorSpeed, LiftConstants.motorSpeed);
+            double clampedError = MathUtil.clamp(error, LiftConstants.backwardsMotorSpeed, LiftConstants.motorSpeed);
+
+            double left = DriveTrain.getInstance().getPitch() > 0 ? power + clampedError : power;
+            double right = DriveTrain.getInstance().getPitch() < 0 ? power - clampedError : power;
+            runLeftUp(left);
+            runRightUp(right);
```

Comment:
I think we need to replace runLeftUp with runLeft and allow it to go up or down from that function, otherwise the PID can only run the motor up